admin-docs/metadata-extraction.md

```diff
@@ -0,0 +1,101 @@
+# Extract metadata information from documents
+
+## Overview
+
+### Introduction
+
+The metadata extraction tool is designed to help you quickly identify, retrieve, and save key details from PDF documents, such as titles, dates, keywords, and other essential information. Powered by machine learning, this tool automates much of the extraction process, saving you time and making it easier to analyze large sets of documents. Whether you’re documenting case studies, managing witness statements, or reviewing legal reports, this tool simplifies data extraction, allowing you to focus on analysis and decision-making.
+
+### How it works
+
+The metadata extraction tool uses machine learning models trained on a vast collection of documents to recognize patterns and extract relevant information. It scans your documents, breaks down the content into smaller segments, like words or phrases, and uses the patterns it learned during training to identify which sections of the document contain useful information. The models learn to do this by being trained on large amounts of sample data, known as training data, which includes examples of documents with labeled information. This allows the models to recognize similar patterns in future documents.
```

Comment:
`smaller segments, like words or phrases` 

Maybe, instead of smaller segments, it could be 'smaller pieces of information'?

Reply:
The word "segments" is intentional:
- Is a broad term that can encompass various levels of document content, from individual words and phrases to larger chunks like sentences or paragraphs
- Machine learning models often process text by breaking it down into smaller, manageable parts. "Segments" effectively conveys this idea of breaking content into smaller units (text segmentation) without overloading the reader with technical details.

I would like to keep it as it is.